packages/hdwallet-native-vault/src/index.test.ts

```diff
@@ -98,7 +98,7 @@ function testVaultImpl(name: string, Vault: ISealableVaultFactory<IVault>) {
       const mnemonic = (await vault.get("#mnemonic")) as native.crypto.Isolation.Engines.Default.BIP39.Mnemonic;
       expect(mnemonic).toBeInstanceOf(native.crypto.Isolation.Engines.Default.BIP39.Mnemonic);
       expect(
-        Buffer.from((await (await mnemonic.toSeed()).toMasterKey()).publicKey).toString("hex")
+        Buffer.from(await (await (await mnemonic.toSeed()).toMasterKey()).getPublicKey()).toString("hex")
```

Comment:
I'd almost rather see chained .then calls than await await await.

Reply:
something like `await mnemonic.toSeed().then(x => x.toMasterKey()).then(x => x.getPublicKey())`? I could see that

Reply:
resolved as e.g.
 ```
expect(
  await mnemonic.toSeed().then(x => x.toMasterKey()).then(x => x.getPublicKey()).then(x => Buffer.from(x).toString("hex"))
).toMatchInlineSnapshot();
```